firmware/src/constants.h

```diff
@@ -88,25 +88,75 @@
  * @brief The Ja Rule message commands.
  */
 typedef enum {
-  ECHO = 0x80,  //!< Echo the data back
-  TX_DMX = 0x81,  //!< Transmit a DMX frame
+  ECHO = 0x80,  //!< Echo the data back. See @ref message_cmd_echo
+
+  TX_DMX = 0x81,  //!< Transmit a DMX frame. See @ref message_cmd_txdmx.
   GET_LOG = 0x82,  //!< Fetch more log data
   GET_FLAGS = 0x83,  //!< Get the flags state
   WRITE_LOG = 0x84,
+
+  /**
+   * @brief Reset the device.
+   * See @ref message_cmd_reset.
```

Comment:
The enums are one per line. If I add @see it breaks the second sentence out into a new paragraph and the table gets much bigger.